### ISSUE TITLE
[Adaptive Cards- Import- Weather Card]: Talkbak does not announce complete information about each day(Thursday, Friday) on Weather card.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7263]
https://microsoft.visualstudio.com/OS/_workitems/edit/34110300

**Comment by Raisul123** [comment]
As per new guideline we are enhancing the severity. so we are closing this issue and logged new bug: [[Adaptive Cards- Import- Weather Card]: Talkbak does not announce complete information about each day(Thursday, Friday) on Weather card. · Issue #8472 · microsoft/AdaptiveCards (github.com)](https://github.com/microsoft/AdaptiveCards/issues/8472)
